labs/lab-1/FillAllPotholes.k

```diff
@@ -0,0 +1,31 @@
+function FillPotHoles () { 
+	while (frontIsClear () ) { 
+		fillPotHole () ;
```

Comment:
See if you can refactor this code so that `fillPotHole` is only ever called when you're actually in position to fill a hole and that work can be done.  Right now your program works but it is making several calls to `fillPotHole` that don't actually do anything.